### PR TITLE
[PREVIEW] Saat

### PR DESCRIPTION
**Pull request by linusnorton**
dockered idam, SIDAM compatible functional tests, SAAT build pipeline

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -111,10 +111,14 @@ dependencies {
     compile "org.hibernate:hibernate-envers"
     compile "org.hibernate.validator:hibernate-validator"
     compile group: 'uk.gov.hmcts.reform.auth', name: 'auth-checker-lib', version: '2.1.3'
+    compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: '5.0.1'
+    compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: '5.0.1'
```
Comment (JeroenRijks) [inline build.gradle]: 3.0.1 passes, 5.0.1 fails for stitching api and NPA

File: infrastructure/output.tf [hunk 2]
```diff
@@ -8,26 +8,18 @@ output "vaultName" {
   value = "${module.local_key_vault.key_vault_name}"
 }
 
-output "idam_api_url" {
+output "idam_api_base_uri" {
   value = "${var.idam_api_url}"
```
Comment (JeroenRijks) [inline infrastructure/output.tf]: worth changing the variables.tf name to idam_api_base_url too?

File: build.gradle (hunk 1 above)
Comment (linusnorton) [reply build.gradle]: seems to work for me

File: infrastructure/output.tf (hunk 2 above)
Comment (linusnorton) [reply infrastructure/output.tf]: yes

**Pull request merged**